workers/unity/Packages/com.improbable.gdk.playerlifecycle/PlayerLifecycleEntityBuilderHelper.cs

```diff
@@ -0,0 +1,16 @@
+using Improbable.Gdk.Core;
+using Improbable.PlayerLifecycle;
+
+namespace Improbable.Gdk.PlayerLifecycle
+{
+    public class PlayerLifecycleEntityBuilderHelper
+    {
+        public static void AddComponents(EntityBuilder entityBuilder, string clientAccess, string serverAccess)
```

Comment:
Should they return the `entityBuilder` they received? It fits with the builder pattern better.

e.g. 
```
return entityBuilder.AddComponent(xyz)
    .AddComponent(abc);
```